core/storage/src/trie/mod.rs

```diff
@@ -391,41 +391,39 @@ impl Trie {
         hash: Option<CryptoHash>,
         partial: NibbleSlice,
         death_row: &mut HashMap<CryptoHash, u32>,
-    ) -> Result<Option<TrieNode>, String> {
+    ) -> Result<(Option<TrieNode>, bool), String> {
```

Comment:
Would be nice to add a comment to delete about return value. It's not clear what bool means, especially since there is an option as well.